packages/core/mixins/_helpers.scss

```diff
@@ -120,6 +175,58 @@
     input[type="button"]::-moz-focus-inner {
         outline: 0;
     }
+
```

Comment:
Jeg lurer på om vi burde se på hele `reset-outline`-mixinen på nytt. Den legger til _veldig mye_ CSS i _veldig mange_ komponenter, og jeg tror ikke alt trengs. Da blir det mindre å "undo" for Forced Colors også..

Reply:
Ser ut som den var en "global" reset på et tidspunkt, for det var ganske mye der ja når jeg så nærmere på den 😅 Forenklet den en del nå, og fra stikkprøvene mine ser det ut som det oppfører seg riktig fremdeles i Safari, Chrome og Firefox.